integration/integration_tms_export_test.go

```diff
@@ -12,11 +12,13 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+const testDockerImage = "devxci/mbtci-java21-node22"
+
 func TestTmsExportIntegrationYaml(t *testing.T) {
 	// success case: run with custom config
 	readEnv()
 	container := givenThisContainer(t, IntegrationTestDockerExecRunnerBundle{
-		Image:       "devxci/mbtci-java11-node14",
```

Comment:
@gerstneralex @artembannikov Any reason you choose this docker image? Does it even matter which docker image you use? You might as well use alpine node or something, right? The TMS code is API calls only, it does not need a toll such as cf, right?